sympy/solvers/solveset.py

```diff
@@ -1061,6 +1062,7 @@ def linear_eq_to_matrix(equations, *symbols):
     row_no = 1
 
     for equation in equations:
+        equation = ratsimp(equation)
         f = sympify(equation)
```

Comment:
There is no need for `sympify` after `ratsimp` -- the return value will be sympified. so `f = ratsimp(equation)` would be fine (provided `equation` is not referrenced again; and if it is, then `f = equation = ratsimp(equation)` would suffice).